src/components/CardBook.js

```diff
@@ -104,12 +103,17 @@ const CardBook = (props) => {
       <Container>
         <div style={{ textAlign: 'left' }}>
           <LightFont bold>{category}</LightFont>
-          <Title>{title}</Title>
+          <Title>{name}</Title>
           <Author>{author}</Author>
         </div>
         <div>
           <Button>Comments</Button>
-          <Button type="button" onClick={() => dispatch(removeBook(id))}>
+          <Button
+            type="button"
+            onClick={() => {
+              dispatch(removeBook(id));
+            }}
+          >
```

Comment:
- [ ] Please add the author as requested by the design given. Please look at the comparison below 👇 
**Your design**
![](https://user-images.githubusercontent.com/83048208/201466740-d37341a2-a3d5-4902-a277-4dc39d3ac812.png)
**Given design**
![](https://user-images.githubusercontent.com/83048208/201466757-e8021c7b-11f3-4d23-aee8-09f5fd83b422.png)

Reply:
Check my pull request. It's working. You probably did to run it properly. Take caution with details please .